api/app/models/auth_token.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class AuthToken < ApplicationRecord
+  include JsonWebToken
+
+  belongs_to :user
+  validates_presence_of :user
+
+  def to_s
+    encode_token(id: id)
+  end
```

Comment:
This feels weird to override a native method, but SO says it's kosher. Maybe just a Ruby convention that differs from JS?

Reply:
This is ok only when you have a strong case for it.  rails does it a lot in actionpack for niceties ( try like 'foo_bar'.humanize ).  If you can manage to do this through adding a method like `as_token` then you will be able to explicitly call that in places and grep your code in the future.  

to_s is called in a lot of implicit places that can obfuscate this

Reply:
ah that makes sense from a debugging perspective. I refactored this to a `to_jwt` method instead and now call it explicitly. From an ease of debugging perspective, would it make sense to override `to_s` to simply call `to_jwt` that way you get `to_jwt` in the stack trace, but also are able to implicitly get the jwt string when casting the instance as a string?